src/libuvc/libuvc.cpp

```diff
@@ -385,6 +374,22 @@ namespace librealsense
                         throw linux_backend_exception(
                                 "Could not open device.");
                     }
+
+                    for(auto ct = uvc_get_input_terminals(_device_handle);
+                        ct; ct = ct->next) {
+                        _input_terminal = ct->bTerminalID;
```

Comment:
What happens if there are multiple terminals - seem that the variable will be overwritten. Is this the expected behaviour? 
Also in case there are no CT defined, the variable shall be initialized elsewhere, probably in the Ctor

Reply:
yes. From what I understand the last one should be used.